docs/src/development/logs.md

```diff
@@ -1,89 +1,127 @@
+## Activity logs
 
-Alternatively, they may also be accessed remotely using the <code>platform log</code> command.  The CLI lets you specify which log file to access (the name of the file below minus the  `.log` extension, as well as view the entire file in a pager, only the most recent lines, and so forth.
+Changes to your environments, such as deployments, cron jobs, and code or variable updates,
+are each logged as activities.
+You can access the logs either in the console or using the CLI:
 
-Run `platform log --help` for complete documentation.
+{{< codetabs >}}
+---
+title=In the console
+highlight=false
+file=none
+---
+
+1. Open the project you are interested in.
+1. Select an environment from the **Environment** menu.
+1. Click a recent activity from the activity feed or click **All** to see the complete history.
 
-A number of different log files are available depending on the application container in use.
+![An activity feed within an environment](/images/management-console/activity.png "0.5")
 
-Although the files in `/var/log` are writable, they should not be written to directly. Only write to it via standard logging mechanisms, such as your application's logging facility.  If your application has its own logging mechanism that should be written to a dedicated logs mount in your application.
+<--->
+---
+title=Using the CLI
+highlight=false
+file=none
+---
 
-All log files are trimmed to 100 MB automatically. But if you need to have complete logs, you can set up cron which will upload them to third-party storage. [Contextual Code](https://www.contextualcode.com/) made a [basic and well-described example](https://gitlab.com/contextualcode/platformsh-store-logs-at-s3) how to achieve it.
+1. Get a list of activities by running `platform act -e <ENVIRONMENT_NAME>`.
+   Pass the `--start` flag to get activities from a specific date in the past.
+1. To see details about the activity's state and timing, run `platform activity:get <ACTIVITY_ID>`,
+   where `<ACTIVITY_ID>` comes from the list in step 1.
+1. Get a log of any given activity by running `platform activity:log <ACTIVITY_ID>`.
 
-## `access.log`
+{{< /codetabs >}}
 
-This is the raw access log for the nginx instance running on the application container. That is, it does not include any requests that return a redirect or cache hit from the [router](/configuration/routes/_index.md).
+### Sharing activity logs
 
-## `app.log`
+When trying to identify and resolve failures, it can often help to have another person's perspective.
+Troubleshoot collaboratively by sharing the logs of specific activities.
 
-Any log messages generated by the application will be sent to this file.  That includes language errors such as PHP Errors, Warnings, and Notices, as well as uncaught exceptions.
+To share a log, open the log and click {{< icon share >}} **Copy URL**.
 
-## `cron.log`
+Share specific lines in the log on clicking the line number.
+Hold shift to select multiple lines.
```

Comment:
```suggestion
Hold <kbd>shift</kbd> to select multiple lines.
```
We might want to use the [kbd tag](https://www.w3schools.com/Tags/tag_kbd.asp) for readility? (even though it's against the markdown guidelines).

Reply:
I'd go with the [Google recommendation](https://developers.google.com/style/ui-elements?hl=en#keyboard) for now until we establish a separate standard. They recommend backticks.

Reply:
And if we do switch, I'd change the default styling :see_no_evil:
![Screenshot from 2022-02-14 09-37-53](https://user-images.githubusercontent.com/26428882/153828734-48305f33-59ac-4052-8709-9f4e54baaa1d.png)